### PR TITLE
Add Event detail section for each event data

### PR DESCRIPTION
**Pull request by priyesh777**
Addresses: https://github.com/idmc-labs/Helix2.0/issues/148#issue-925460022

## Depends on:
https://github.com/idmc-labs/helix-server/pull/195#issue-687284912

## Changes
- Event Detail section is added for each event of the event-list table that shows quick info about few details of the event. However, more details about the event may be required.

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [x] permission checks
- [x] translations

### CODE REVIEW

File: src/views/Event/index.tsx [hunk 1]
```diff
@@ -68,6 +69,29 @@ function Event(props: EventProps) {
...
+            <Container
+                className={styles.container}
+                heading="Event Details"
+            >
+                {eventData ? (
+                    <div className={styles.stats}>
+                        <TextBlock
+                            label="Event ID"
+                            value={eventData?.event?.id}
+                        />
```
Comment (tnagorra) [inline src/views/Event/index.tsx:81]: We don't need to show event id

File: src/views/Event/index.tsx [hunk 2]
```diff
@@ -68,6 +69,29 @@ function Event(props: EventProps) {
...
+                        <TextBlock
+                            label="Event Name"
+                            value={eventData?.event?.name}
+                        />
+                        <TextBlock
+                            label="Crisis"
+                            value={eventData?.event?.crisis?.name}
+                        />
+                    </div>
```
Comment (tnagorra) [inline src/views/Event/index.tsx:90]: We need to show more information

File: src/components/TextBlock/index.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+                {label}
+            </div>
+            <div className={styles.value}>
+                {value ?? 'N/a'}
+            </div>
+        </div>
+    );
+}
+
+export default NumberBlock;
```
Comment (tnagorra) [inline src/components/TextBlock/index.tsx:27]:
```suggestion
function TextBlock({
    label,
    value,
    className,
}: {
    label: string;
    value: string | null | undefined;
    className?: string;
}) {
    return (
        <div className={_cs(styles.textBlock, className)}>
            <div className={styles.label}>
                {label}
            </div>
            <div className={styles.value}>
                {value ?? 'N/a'}
            </div>
        </div>
    );
}

export default TextBlock;
```

**Review by tnagorra — CHANGES_REQUESTED** [review]

**Review by tnagorra — CHANGES_REQUESTED** [review]
Doesn't work with develop branch.
Please add the dependency with the server.

File: src/views/Country/index.tsx [hunk 4]
```diff
@@ -258,37 +258,83 @@ function Country(props: CountryProps) {
                         >
                             <div className={styles.stats}>
                                 <NumberBlock
-                                    label={`New Displacements (Conflict ${year})`}
+                                    label={(
+                                        <>
+                                            New Displacements
+                                            <br />
+                                            {`(Conflict ${year})`}
```
Comment (AdityaKhatri) [inline src/views/Country/index.tsx:265]: Make a subLabel prop

**Review by AdityaKhatri — APPROVED** [review]

**Pull request merged**